src/cli/repo/repo.go

```diff
@@ -60,6 +60,7 @@ func AddCommands(cmd *cobra.Command) {
 	flags.AddForceMaintenanceFlag(maintenanceCmd)
 	flags.AddMaintenanceUserFlag(maintenanceCmd)
 	flags.AddMaintenanceHostnameFlag(maintenanceCmd)
+	flags.AddCorsoUpdatePassphraseFlags(updatePassphraseCmd)
```

Comment:
We should make the flag required as there is no point in the user calling without it.

PS: I don't think we need the word `Corso` in the function name, but we'll keep it since it is already there anyways.

Reply:
We were using the Corso word everywhere. But since its removed from other places it makes sense to remove it from here too.
Always added flag to required.